.github/workflows/profiling.yaml

```diff
@@ -40,78 +27,58 @@ jobs:
           echo "Context: "
           cat ${GITHUB_EVENT_PATH}
 
-          echo "Creating Download URL from ISSUE"
-          if ! echo "DOWNLOAD_URL=$(cat ${GITHUB_EVENT_PATH} | jq '.comment.body' | grep -oP '(https:)(.*?)(?=\))' )" >> ${GITHUB_ENV}; then
-            echo "Must be an issue body"
-          else
-            echo "DOWNLOAD_URL=$(cat ${GITHUB_EVENT_PATH} | jq '.issue.body' | grep -oP '(https:)(.*?)(?=\))' )" >> ${GITHUB_ENV}
-            echo "Found it in the issue body"
-          fi 
-          
-          echo "GITHUB_ISSUE_ID=$(cat ${GITHUB_EVENT_PATH} | jq '.issue.number')" >> ${GITHUB_ENV}
-          echo "FILE_NAME=profiling-data.tar.gz" >> ${GITHUB_ENV}
-
-      - name: Download Files & Test
+  Profiling:
+    needs: Prepare
+    runs-on: [Linux, performance]
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Clean _work Folder
         run: |
-          set -euo pipefail
-          echo "${DOWNLOAD_URL}"
-          curl -LO ${DOWNLOAD_URL}
-          echo "FILE_NAME=$(basename ${DOWNLOAD_URL})" >> ${GITHUB_ENV}
-          file $(basename ${DOWNLOAD_URL})
-
+          echo "Cleaning Folder"
+          rm -rf ./*
       - name: Run Tests
         run: |
           set -euo pipefail
+          ISSUE_NUMBER=$(cat ${GITHUB_EVENT_PATH} | jq '.issue.number')
+          gh issue comment ${ISSUE_NUMBER} --body "IN PROGRESS: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          DOWNLOAD_URL=$(cat ${GITHUB_EVENT_PATH} | jq '.comment.body' | grep -oP '(https:)(.*?)(?=\))' )
+          FILE_NAME=$(basename ${DOWNLOAD_URL})
+          echo "Download Test File"
+          curl -LOsS ${DOWNLOAD_URL}
```

Comment:
See above, we could check whether this is going to work on the flyweight executor (and not send a message if it wouldn't).